### PR TITLE
Add examples for debug

### PR DESCRIPTION
**Pull request by 1e-to**

### CODE REVIEW

File: numba_dppy/examples/debug/dppy_func.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    a = np.array(np.random.random(N), dtype=np.float32)
+    b = np.array(np.random.random(N), dtype=np.float32)
+    c = np.ones_like(a)
+
+    context = get_context()
+
+    print("Device Context:", context)
+
+    with dpctl.device_context(context):
```
Comment (PokhodenkoSA) [inline numba_dppy/examples/debug/dppy_func.py:65]: Use default context and provide info in example description about possibility to use env variable for changing default device context.
Also description should contain info about how to run example.

File: numba_dppy/examples/debug/dppy_func.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+def main():
+    global_size = 10
+    N = global_size
+    print("N", N)
+
+    a = np.array(np.random.random(N), dtype=np.float32)
+    b = np.array(np.random.random(N), dtype=np.float32)
+    c = np.ones_like(a)
```
Comment (PokhodenkoSA) [inline numba_dppy/examples/debug/dppy_func.py:59]:
```suggestion
    c = np.empty_like(a)
```

File: numba_dppy/examples/debug/dppy_func.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    return device
+
+
+def main():
+    global_size = 10
+    N = global_size
+    print("N", N)
+
+    a = np.array(np.random.random(N), dtype=np.float32)
+    b = np.array(np.random.random(N), dtype=np.float32)
```
Comment (PokhodenkoSA) [inline numba_dppy/examples/debug/dppy_func.py:58]:
```suggestion
    a = np.arange(N, dtype=np.float32)
    b = np.arange(N, dtype=np.float32)
```

File: numba_dppy/examples/debug/dppy_func.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+import numpy as np
+import numba_dppy as dppy
+import dpctl
+
+
+@dppy.func
+def func_sum(a_in_func, b_in_func):
+    s = a_in_func + b_in_func
+    return s
```
Comment (akharche) [inline numba_dppy/examples/debug/dppy_func.py:23]:
```suggestion
def func_sum(a_func_arg, b_func_arg):
    local_var = a_func_arg + b_func_arg
    return local_var
```

File: numba_dppy/examples/debug/dppy_func.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+
+@dppy.kernel
+def kernel_sum(a_in_kernel, b_in_kernel, c_in_kernel):
+    i = dppy.get_global_id(0)
+    c_in_kernel[i] = func_sum(a_in_kernel[i], b_in_kernel[i])
```
Comment (akharche) [inline numba_dppy/examples/debug/dppy_func.py:29]:
```suggestion
def kernel_sum(a_kernel_arg, b_kernel_arg, c_kernel_arg):
    id = dppy.get_global_id(0)
    c_kernel_arg[id] = func_sum(a_kernel_arg[id], b_kernel_arg[id])
```

**Comment by 1e-to** [comment]
Fixed

File: numba_dppy/examples/debug/dppy_func.py (hunk 4 above)
Comment (PokhodenkoSA) [inline numba_dppy/examples/debug/dppy_func.py:23]: @1e-to 
`s` to `result`?

**Review by PokhodenkoSA — APPROVED** [review]

**Comment by PokhodenkoSA** [comment]
Tested locally.

**Pull request merged**